Sources/Tropos/Views/DailyForecastView.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import TroposCore
+
+@objc(TRDailyForecastView) public class DailyForecastView: UIView {
+    @IBOutlet weak var dayOfWeekLabel: UILabel!
+    @IBOutlet weak var highTemperatureLabel: UILabel!
+    @IBOutlet weak var lowTemperatureLabel: UILabel!
+    @IBOutlet weak var conditionsImageView: UIImageView!
+    @IBOutlet var contentView: DailyForecastView!
+    @objc public var viewModel: DailyForecastViewModel!
```

Comment:
While it’s useful for the outlets to be implicitly unwrapped optionals (because they’re set automatically when the view is loaded), I think it would make sense for this `viewModel` property to be a regular optional.